tasks/tests/unit/test_test_results_finisher.py

```diff
@@ -446,7 +446,7 @@ def test_upload_finisher_task_call(
 </details>
 
 To view more test analytics, go to the [Test Analytics Dashboard](https://app.codecov.io/gh/test-username/test-repo-name/tests/main)
-Got feedback? Let us know on [Github](https://github.com/codecov/feedback/issues)""",
+Got feedback? Let us know on [Github](https://github.com/codecov/feedback/issues/304/304)""",
```

Comment:
```suggestion
Got feedback? Let us know on [Github](https://github.com/codecov/feedback/issues/304)""",
```

I think there was a typo